cpp/cmake/find_protobuf.cmake

```diff
@@ -0,0 +1,2 @@
+find_package(Protobuf REQUIRED)
```

Comment:
Should it place under `cpp/cmake/Modules/` according to 
https://github.com/pingcap/tipb/blob/e0e0a3a17646e480b25f2f9cdc1763023f34310b/cpp/CMakeLists.txt#L15